samples/AnomalyDetector/src/Generated/Models/MultivariateDetectionResult.Serialization.cs

```diff
@@ -101,7 +115,14 @@ internal static MultivariateDetectionResult DeserializeMultivariateDetectionResu
                     List<AnomalyState> array = new List<AnomalyState>();
                     foreach (var item in property.Value.EnumerateArray())
                     {
-                        array.Add(AnomalyState.DeserializeAnomalyState(item));
+                        if (item.ValueKind == JsonValueKind.Null)
+                        {
+                            array.Add(null);
+                        }
+                        else
+                        {
+                            array.Add(AnomalyState.DeserializeAnomalyState(item));
+                        }
```

Comment:
this is a change coming from symmetry between serialization and deserialization.
Before this change, we also add null - the `AnomalyState.DeserializeAnomalyState` method will return null value if `item.ValueKind` is null, this change only takes that part out here.